core/templates/wikified-tiddler-visible-code.tid

```diff
@@ -0,0 +1,3 @@
+title: $:/core/templates/wikified-tiddler-visible-code
```

Comment:
I thought I had a good reason for making the new template, but I have no idea what it was. So yes, it should just reuse $:/core/templates/html-tiddler